__tests__/news-server.test.js

```diff
@@ -32,13 +32,40 @@ describe("GET /api/topics", () => {
         });
       });
   });
-
   test("404: responds with an error message when given a path that does not exist", () => {
     return request(app)
       .get("/api/notAPath")
       .expect(404)
       .then(({ body }) => {
-        expect(body.msg).toBe("Error: path does not exist");
+        expect(body.msg).toBe("Error: 404 not found");
+      });
+  });
+});
+
+describe("GET /api/articles", () => {
+  test("200: responds with an article selected by its id, inputted by the user", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
```

Comment:
You need to check for the id you've asked for - 1 in this case.
At the moment, you can't be sure you've got back the correct article